detect.go

```diff
@@ -28,18 +25,10 @@ func Detect(parser ConfigurationParser) packit.DetectFunc {
 			},
 		}
 
-		shouldEnableReload, err := checkLiveReloadEnabled()
-		if err != nil {
+		if shouldEnableReload, err := reload.ShouldRequireWatchexec(); err != nil {
```

Comment:
personal style opinion:

I prefer to see:

```
x, err:=foo()
if err != nil {
  return err
}

if x {
   bar()
}
```

rather than:

```
if x, err := foo(); err != nil{
  return err
} else if x {
  bar()
}
```

But that's a stylistic thing - I wouldn't object to either styles